docs/assets/javascripts/version-selector.js

```diff
@@ -0,0 +1,167 @@
+window.addEventListener("DOMContentLoaded", function() {
+    window.versionPages = {};
+    var VERSION = window.location.pathname.split("/")[1];
+    var VERSION_LATEST = ".";
+
+    function populateVersionSitemap(version) {
+        var versionPath = version === VERSION_LATEST ? "" : "/" + version;
+        window.versionPages[version] = [];
+
+        var xhrSitemap = new XMLHttpRequest();
+        var sitemapURL = window.location.origin + versionPath + '/sitemap.xml';
+        xhrSitemap.open("GET", sitemapURL);
+        xhrSitemap.onload = function() {
+            var xmlLoc = this.responseXML.getElementsByTagName("loc");
+            var nodeText = [];
+    
+            for (var index = 0; index < xmlLoc.length; index++) {
+                var element = xmlLoc[index];
+                nodeText.push(element.textContent);
+            }
+            var prefix = nodeText[0].slice(0,-1);
+            window.versionPages[version] = nodeText.map(function(e) {
+                return removePrefix(e, prefix);
+            });
+        };
+        xhrSitemap.send();
+    }
+
+    function removePrefix(str, prefix) {
+        var hasPrefix = str.indexOf(prefix) === 0;
+        return hasPrefix ? str.substr(prefix.length) : str.toString();
+    }
+
+    function makeSelect(options, selected) {
+        var select = document.createElement("select");
+        select.classList.add("select-css");
+        var deprecatedVersionsArray = new Array();
+        var versionsArray = new Array();
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/codacy/docs/pullRequest?prid=26905)
```suggestion
        container.className = "version-select-container";
```